### PR TITLE
Update documentation to match the consistent Remixer Docs structure

### PR DESCRIPTION
**Pull request by miguelandres**
- Replaced logo with a smaller version embedded in an H1.
- Added screenshot to the right.
- Renamed _Getting started_ to _Using Remixer_
- Added sections in `README.md` for _Using Remixer_ and _Contributing_ instead of them being just parts of a table of contents.
- Adds documentation on how to configure the Firebase Remote Controller from the point of view of the Android Remixer Framework
- Adds an explanation on when you may not want/need a RemixerFragment.

### CODE REVIEW

File: remixer_storage/src/main/java/com/google/android/libraries/remixer/storage/FirebaseRemoteControllerSyncer.java [hunk 1]
```diff
@@ -264,18 +264,25 @@ public void addSharingStatusListener(@NonNull SharingStatusListener listener) {
...
-        remoteId);
+    String url = getRemoteUrl();
     String message =
         applicationContext.getResources().getString(R.string.shareLinkFormat, url);
     intent.putExtra(Intent.EXTRA_TEXT, message);
     intent.setType("text/plain");
     return intent;
   }
 
+  /**
```
Comment (chuga) [inline remixer_storage/src/main/java/com/google/android/libraries/remixer/storage/FirebaseRemoteControllerSyncer.java:275]: Did you mean to add this to this CL?

File: CONFIGURE_FIREBASE.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Configure the Firebase Remote Controller
```
Comment (chuga) [inline CONFIGURE_FIREBASE.md]: nit: Configuring the Firebase Remote Controller

File: CONFIGURE_FIREBASE.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+There are three parts to being able to use the Firebase Remote Controller with Remixer for Android:
```
Comment (chuga) [inline CONFIGURE_FIREBASE.md]: nit: r/parts/steps

File: CONFIGURE_FIREBASE.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+1. Install the Firebase Remote Controller on a Firebase Instance. This is out of the scope of this document, as it is properly documented [in the remote's repo](https://www.github.com/material-foundation/material-remixer-remote-web).
+2. Get the proper google-services.json from the [Firebase console](https://console.firebase.google.com) file and put it in your app's module.
+3. Set up Remixer synchronization to use the Firebase Remote.
+
+## google-services.json
```
Comment (chuga) [inline CONFIGURE_FIREBASE.md]: It took me a while to understand that this is the content for step 2. I'd suggest prepending these headers with the step number.

File: CONFIGURE_FIREBASE.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+# Configure the Firebase Remote Controller
+
+There are three parts to being able to use the Firebase Remote Controller with Remixer for Android:
+
+1. Install the Firebase Remote Controller on a Firebase Instance. This is out of the scope of this document, as it is properly documented [in the remote's repo](https://www.github.com/material-foundation/material-remixer-remote-web).
+2. Get the proper google-services.json from the [Firebase console](https://console.firebase.google.com) file and put it in your app's module.
```
Comment (chuga) [inline CONFIGURE_FIREBASE.md]: Since you're explaining what to do in the following subsections I'd just name the steps here. In this case, I'd change this to just "Step 2: Set up the Firebase credentials in your app". Step 3 looks good, for Step 1 I'd put the "This is out of scope" message in a separate section like you do for the other 2 steps.

File: CONFIGURE_FIREBASE.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+- Click _Add another app_ and follow the instructions (you can reuse an already registered app if that is appropriate).
+- At the end you'll download a google-services.json file. Put that next to your application module's `build.gradle`
+
+More detail can be found in [the Firebase documentation](https://firebase.google.com/docs/android/setup#manually_add_firebase).
+
+## Set up Remixer synchronization
+
+Remixer relies on implementations of `SynchronizationMechanism` to store and sync values. This is set on your Application class's `onCreate()` method.
+
+In order to use the Remote you only need to use the right one, `FirebaseRemoteControllerSyncer`, and optionally force it to start sharing variable status immediately
```
Comment (chuga) [inline CONFIGURE_FIREBASE.md]: What do you mean by the 'right' one? Is there more than one?

File: CONFIGURE_FIREBASE.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+import com.google.android.libraries.remixer.ui.RemixerInitialization;
+
+class MyApplication extends Application {
+  @Override
+  public void onCreate() {
+    super.onCreate();
+    RemixerInitialization.initRemixer(this);
+    FirebaseRemoteControllerSyncer syncer = new FirebaseRemoteControllerSyncer(this);
+    Remixer.getInstance().setSynchronizationMechanism(syncer);
+    // Optionally you can start sharing your variables to the Remote automatically.
```
Comment (chuga) [inline CONFIGURE_FIREBASE.md:40]: What happens if you don't start sharing? Does this replace the toggle in the UI? If so it would be good to explain it here

File: CONFIGURE_FIREBASE.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+    syncer.startSharing();
+  }
+}
+```
+
+Since all the Variable Adjustment UI is completely available in the Firebase Remote Controller, if you start sharing in the Application class you may not want/need to use the `RemixerFragment` at all. In that case you can use `FirebaseRemoteControllerSyncer#getShareLinkIntent()` or `FirebaseRemoteControllerSyncer#getRemoteUrl()` to direct users to the remote on your own terms.
+
+Otherwise, the `RemixerFragment` will automatically show options to share variables with the remote controller and share the link.
```
Comment (chuga) [inline CONFIGURE_FIREBASE.md]: If you can add a screenshot here of the sharing UI you're talking about. It will make it easier to understand for someone who hasn't seen the UI yet

File: README.md [hunk 9]
```diff
@@ -1,49 +1,40 @@
-![Remixer](https://cdn.rawgit.com/material-foundation/material-remixer/master/docs/assets/lockup_remixer_icon_horizontal_dark_small.svg)
+# ![Remixer](small_logo.png) Remixer Android
+<img align="right" src="remixer_screenshot.png" width="300px">
 
 [![TravisCI Build Status](https://travis-ci.org/material-foundation/material-remixer-android.svg?branch=develop)](https://travis-ci.org/material-foundation/material-remixer-android) [![CircleCI Build Status](https://circleci.com/gh/material-foundation/material-remixer-android.svg?style=svg)](https://circleci.com/gh/material-foundation/material-remixer-android) [![codecov](https://codecov.io/gh/material-foundation/material-remixer-android/branch/develop/graph/badge.svg)](https://codecov.io/gh/material-foundation/material-remixer-android)
 
-# Remixer Android
+Remixer is a framework to iterate quickly on UI changes by allowing you to adjust UI variables without needing to rebuild (or even restart) your app. You can adjust Numbers, Colors, Booleans, and Strings; check out the [example app](https://github.com/material-foundation/material-remixer-android/tree/develop/remixer_example)
```
Comment (chuga) [inline README.md]: I'd change "; check out the [example app]" to ". To see it in action check out the [example app]"

File: README.md [hunk 10]
```diff
@@ -1,49 +1,40 @@
...
-1. [Getting started](GETTING_STARTED.md)
-2. [Configure the UI](CONFIGURE_UI.md)
-3. [Project structure](PROJECT_STRUCTURE.md)
-4. [Supported Data Types](SUPPORTED_DATA_TYPES.md)
-5. [Extending Remixer](EXTENDING_REMIXER.md)
-6. API Reference (_notice this includes internal APIs_):
-  - [Current Version 0.6.6](https://jitpack.io/com/github/material-foundation/material-remixer-android/remixer/0.6.6/javadoc/index.html)
-  - [Develop Snapshot](https://jitpack.io/com/github/material-foundation/material-remixer-android/remixer/develop-SNAPSHOT/javadoc/index.html)
+- [Using Remixer](USING_REMIXER.md)
+  - [Configure the UI](CONFIGURE_UI.md)
```
Comment (chuga) [inline README.md]: These shouldn't be subsections of the doc linked above, right?

File: README.md [hunk 11]
```diff
@@ -1,49 +1,40 @@
...
 
-- - -
+## Contributing to Remixer
 
-## State of development
+We're excited you want to contribute to the project! Please read these docs if you're interested so we can get your contributions submitted quickly.
 
-Visit our [State of Development](https://github.com/material-foundation/material-remixer/wiki/State-of-Development) wiki for the current roadmap and status of development for each platform.
+- [Contribution policy and guidelines](CONTRIBUTING.md)
+  - [Project structure](PROJECT_STRUCTURE.md)
```
Comment (chuga) [inline README.md]: Same here, fix indentation

File: README.md [hunk 12]
```diff
@@ -1,49 +1,40 @@
...
+- [Using Remixer](USING_REMIXER.md)
+  - [Configure the UI](CONFIGURE_UI.md)
+  - [Supported Data Types](SUPPORTED_DATA_TYPES.md)
+  - (Optionally) [Configure the Firebase Remote Controller](CONFIGURE_FIREBASE.md)
 
-- - -
+## Contributing to Remixer
 
-## State of development
+We're excited you want to contribute to the project! Please read these docs if you're interested so we can get your contributions submitted quickly.
```
Comment (chuga) [inline README.md]: These two sentences are a little contradictory :) I'd remove the "if you're interested" from the second one as the first one assumes you are.

**Review by chuga — COMMENTED** [review]
Mostly nits

File: remixer_storage/src/main/java/com/google/android/libraries/remixer/storage/FirebaseRemoteControllerSyncer.java (hunk 1 above)
Comment (miguelandres) [reply remixer_storage/src/main/java/com/google/android/libraries/remixer/storage/FirebaseRemoteControllerSyncer.java:275]: Actually yes, it was a tiny thing that was missing and I noticed in documentation, I meant to offer this from the beginning. Writing another CL seemed like overdoing it.

File: CONFIGURE_FIREBASE.md (hunk 2 above)
Comment (miguelandres) [reply CONFIGURE_FIREBASE.md]: Done

File: CONFIGURE_FIREBASE.md (hunk 3 above)
Comment (miguelandres) [reply CONFIGURE_FIREBASE.md]: Done

File: remixer_storage/src/main/java/com/google/android/libraries/remixer/storage/FirebaseRemoteControllerSyncer.java (hunk 1 above)
Comment (chuga) [reply remixer_storage/src/main/java/com/google/android/libraries/remixer/storage/FirebaseRemoteControllerSyncer.java:275]: Np, just checking

File: CONFIGURE_FIREBASE.md (hunk 6 above)
Comment (miguelandres) [reply CONFIGURE_FIREBASE.md]: Improved wording.

File: CONFIGURE_FIREBASE.md (hunk 4 above)
Comment (miguelandres) [reply CONFIGURE_FIREBASE.md]: Done

File: CONFIGURE_FIREBASE.md (hunk 7 above)
Comment (miguelandres) [reply CONFIGURE_FIREBASE.md:40]: It is explained below.

File: CONFIGURE_FIREBASE.md (hunk 8 above)
Comment (miguelandres) [reply CONFIGURE_FIREBASE.md]: Agreed, good idea

File: README.md (hunk 12 above)
Comment (miguelandres) [reply README.md]: Done.

File: README.md (hunk 10 above)
Comment (miguelandres) [reply README.md]: Done

File: README.md (hunk 11 above)
Comment (miguelandres) [reply README.md]: Done

File: USING_REMIXER.md [hunk 13]
```diff
@@ -1,10 +1,10 @@
-# Getting started
+# Using Remixer
 
-__Disclaimer:__ Remixer still hasn't reached a stage that we consider is stable enough to commit to the current status of the API, it will be evolving quickly and we may commit breaking changes every once in a while. _That being said_, we would love to have you try it out and tell us what you think is missing and what you'd like us to focus on.
+This page explains what you minimally need to do to get Remixer running on your app. However, if you're the kind of person who learns by example, you may want to look at our [example app](https://github.com/material-foundation/material-remixer-android/tree/develop/remixer_example).
```
Comment (chuga) [inline USING_REMIXER.md]: one last nit: remove 'minimally'

**Review by chuga — APPROVED** [review]

File: USING_REMIXER.md (hunk 13 above)
Comment (miguelandres) [reply USING_REMIXER.md]: Done
